sycl/include/sycl/interop_handle.hpp

```diff
@@ -183,12 +208,25 @@ class interop_handle {
   friend class detail::DispatchHostTask;
   using ReqToMem = std::pair<detail::AccessorImplHost *, ur_mem_handle_t>;
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  // MGraph should become a member of this class on the next ABI breaking
+  // window
+  // TODO create and link GitHub Issue
+  interop_handle(std::vector<ReqToMem> MemObjs,
+                 const std::shared_ptr<detail::queue_impl> &Queue,
+                 const std::shared_ptr<detail::device_impl> &Device,
+                 const std::shared_ptr<detail::context_impl> &Context,
+                 ur_exp_command_buffer_handle_t Graph = nullptr)
+      : MQueue(Queue), MDevice(Device), MContext(Context), MGraph(Graph),
+        MMemObjs(std::move(MemObjs)) {}
+#else
   interop_handle(std::vector<ReqToMem> MemObjs,
                  const std::shared_ptr<detail::queue_impl> &Queue,
                  const std::shared_ptr<detail::device_impl> &Device,
                  const std::shared_ptr<detail::context_impl> &Context)
       : MQueue(Queue), MDevice(Device), MContext(Context),
         MMemObjs(std::move(MemObjs)) {}
+#endif
```

Comment:
```suggestion
  interop_handle(std::vector<ReqToMem> MemObjs,
                 const std::shared_ptr<detail::queue_impl> &Queue,
                 const std::shared_ptr<detail::device_impl> &Device,
                 const std::shared_ptr<detail::context_impl> &Context,
#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
                 [[maybe_unused]]
#endif
                 ur_exp_command_buffer_handle_t Graph = nullptr)
      : MQueue(Queue), MDevice(Device), MContext(Context),
#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
        // MGraph should become a member of this class on the next ABI breaking
        // window
        // TODO create and link GitHub Issue
        MGraph(Graph),
#endif
        MMemObjs(std::move(MemObjs)) {}
```

Maybe a little inelegant, but it lets us use the new ctor pre-emptively without the macro.